nvmw.bat

```diff
@@ -75,27 +81,44 @@ exit /b 0
 :install
 setlocal
 
+set NODE_TYPE=node

```

Comment:
Umm, `move v* node\` is not working for me.

![2015-01-20 15 36 32](https://cloud.githubusercontent.com/assets/346598/5813056/2d09a98e-a0ba-11e4-99c0-dbc17b5fab5f.png)